Volha_Ivanova/Module_1_Java/Java_Fundamentals/HomeWork1/src/com/epam/cdp/volha_ivanova_task1/Runner.java

```diff
@@ -0,0 +1,35 @@
+package com.epam.cdp.volha_ivanova_task1;
+
```

Comment:
For all code please use formatter. Your code style is not uniform
We will use standard formatter for your IDE, in case we don't use special style now. 
I suppose you can google how to use formatter for your IDE